ci-operator/step-registry/openstack/rotate/cloud-credentials/openstack-rotate-cloud-credentials-commands.sh

```diff
@@ -43,16 +43,25 @@ delete_application_credential() {
 info 'Replacing clouds.yaml in the openstack-cloud-credentials secret...'
 oc set data -n kube-system secret/openstack-credentials clouds.yaml="$(<"$ALTERNATIVE_CLOUDS_YAML")"
 
-info 'Waiting for the cluster to become ready...'
-declare progressing=1
-while [[ $progressing -gt 0 ]]; do
-	sleep 5
-	progressing="$(oc get clusteroperator -o json \
-			| jq '.items[].status.conditions[] | select(.type=="Progressing") | select(.status=="True").status' \
-			| wc -l)"
-	info "${progressing} operators progressing."
+sleep 5
```

Comment:
Is the sleep really needed when using `oc wait` ?

Reply:
the operators need time before they realise there's a progression to start